src/shared-components/typography/index.ts

```diff
@@ -75,7 +74,10 @@ const buttonStyle = (theme: ThemeType) => `
   text-transform: uppercase;
 `;
 
-const linkStyle = () => `
+/**
+ * TODO-MA: Add theme.TYPOGRAPHY.fontSize.link to guarantee font-size compatibility
```

Comment:
We should enforce the Figma spec of 14px for links. Not include font-size means it'll be dependent on other styles / every dev implementation. We should provide more batteries included functionality.